code/game/jobs/job/marines.dm

```diff
@@ -21,6 +21,17 @@
 		. = ..()
 		if(flags_startup_parameters & ROLE_ADD_TO_MODE) H.nutrition = rand(60,250) //Start hungry for the default marine.
 
+	generate_wearable_equipment()
```

Comment:
Check the "All BYOND paths must contain the full path" in the guide to contributing: https://github.com/ColonialMarines-Mirror/ColonialMarines-2018/blob/master/CONTRIBUTING.md